### ISSUE TITLE
resource_bundles can't work 

### ISSUE THREAD

**Issue by ZhangYiGang** [CocoaPods/CocoaPods#9310]
# Report

## What did you do?

```ruby
   s.resource_bundles = {   'resourceBundle1' => ['image1/*.png']}
s.subspec 'Core' do |core|
   
   core.resources = ["MatrixKit/**/*.{xib}","MatrixKit/Assets/resourceBundle.bundle", "MatrixKit/Assets/MatrixKitAssets.bundle"]
 end
```
 
i want to change the open source program named Matrixkit,so i add the first line 
```ruby
s.resource_bundles = {   'resourceBundle1' => ['image1/*.png']}
```

but when i execute pod install and build the main program, but i cant't find the  new bundle in   
▸ ⁨用户⁩ ▸ ⁨**▸ ⁨library⁩ ▸ ⁨Developer⁩ ▸ ⁨CoreSimulator⁩ ▸ ⁨Devices⁩ ▸ ⁨B0CC2300-0714-42CC-A875-BBBAFD5A6B1F⁩ ▸ ⁨data⁩ ▸ ⁨Containers⁩ ▸ ⁨Bundle⁩ ▸ ⁨Application⁩ ▸ ⁨E7DFFA07-1398-4F98-B43A-A911A50C4761⁩ ▸ ⁨

## What did you expect to happen?
in the above path,there should have a bundle named resourceBundle1

```
CocoaPods : 1.8.3
        Ruby : ruby 2.2.4p230 (2015-12-16 revision 53155) [x86_64-darwin18]
    RubyGems : 3.0.6
        Host : Mac OS X 10.14.6 (18G95)
       Xcode : 10.2 (10E125)
         Git : git version 2.20.1 (Apple Git-117)
Ruby lib dir : /Users/bupt/.rvm/rubies/ruby-2.2.4/lib
Repositories : master - git - https://github.com/CocoaPods/Specs.git @ fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
HEAD
               trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /Users/bupt/.rvm/gems/ruby-2.2.4/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project
platform :ios, '10.0'

# Use frameforks to allow usage of pod written in Swift (like PiwikTracker)
use_frameworks!


# Different flavours of pods to MatrixKit
# The current MatrixKit pod version
$matrixKitVersion = '0.10.2'

# The develop branch version
#$matrixKitVersion = 'develop'

# The one used for developing both MatrixSDK and MatrixKit
# Note that MatrixSDK must be cloned into a folder called matrix-ios-sdk next to the MatrixKit folder
$matrixKitVersion = 'local'


# Method to import the right MatrixKit flavour
def import_MatrixKit
    if $matrixKitVersion == 'local'
        pod 'MatrixSDK', :path => 'matrix-ios-sdk/MatrixSDK.podspec'
        pod 'MatrixSDK/SwiftSupport', :path => 'matrix-ios-sdk/MatrixSDK.podspec'
        pod 'MatrixSDK/JingleCallStack', :path => 'matrix-ios-sdk/MatrixSDK.podspec'
        pod 'MatrixKit', :path => 'matrix-ios-kit/MatrixKit.podspec'
    else
        if $matrixKitVersion == 'develop'
            pod 'MatrixSDK', :git => 'https://github.com/matrix-org/matrix-ios-sdk.git',
#            :branch =>
#            'v0.13.0'
            :branch => 'develop'
            pod 'MatrixSDK/SwiftSupport', :git => 'https://github.com/matrix-org/matrix-ios-sdk.git',
#            :branch => 'v0.13.0'
:branch => 'develop'

            pod 'MatrixSDK/JingleCallStack', :git => 'https://github.com/matrix-org/matrix-ios-sdk.git',
#            :branch => 'v0.13.0'
:branch => 'develop'

pod 'MatrixKit', :git => 'https://github.com/matrix-org/matrix-ios-kit.git',    :branch => 'develop'

#            :branch => 'v0.10.0'
        else
            pod 'MatrixKit', $matrixKitVersion
            pod 'MatrixSDK/SwiftSupport'
            pod 'MatrixSDK/JingleCallStack'
        end
    end
end

# Method to import the right MatrixKit/AppExtension flavour
def import_MatrixKitAppExtension
    if $matrixKitVersion == 'local'
        pod 'MatrixSDK', :path => 'matrix-ios-sdk/MatrixSDK.podspec'
        pod 'MatrixSDK/SwiftSupport', :path => 'matrix-ios-sdk/MatrixSDK.podspec'
        pod 'MatrixKit/AppExtension', :path => 'matrix-ios-kit/MatrixKit.podspec'
    else
        if $matrixKitVersion == 'develop'
            pod 'MatrixSDK', :git => 'https://github.com/matrix-org/matrix-ios-sdk.git',
#:branch => 'v0.13.0'
:branch => 'develop'

            pod 'MatrixSDK/SwiftSupport', :git => 'https://github.com/matrix-org/matrix-ios-sdk.git',
#            :branch => 'v0.13.0'
:branch => 'develop'

            pod 'MatrixKit/AppExtension', :git => 'https://github.com/matrix-org/matrix-ios-kit.git',
#            :branch => 'v0.10.0'
:branch => 'develop'

        else
            pod 'MatrixKit/AppExtension', $matrixKitVersion
            pod 'MatrixSDK/SwiftSupport'
        end
    end
end


abstract_target 'RiotPods' do

    pod 'GBDeviceInfo', '~> 5.2.0'
    pod 'Reusable', '~> 4.1'
    pod 'SwiftUTI', :git => 'https://github.com/speramusinc/SwiftUTI.git', :branch => 'master'

    # Piwik for analytics
    pod 'MatomoTracker', '~> 6.0.1'

    # Remove warnings from "bad" pods
    pod 'OLMKit', :inhibit_warnings => true
    pod 'cmark', :inhibit_warnings => true
    pod 'DTCoreText', :inhibit_warnings => true
    pod 'zxcvbn-ios'

    # Tools
    pod 'SwiftGen', '~> 6.1'
    pod 'SwiftLint', '~> 0.33.0'

    target "Riot" do
        import_MatrixKit
        pod 'DGCollectionViewLeftAlignFlowLayout', '~> 1.0.4'

        target 'RiotTests' do
            inherit! :search_paths
        end
    end

    target "RiotShareExtension" do
        import_MatrixKitAppExtension
    end

    target "SiriIntents" do
        import_MatrixKitAppExtension
    end

end


post_install do |installer|
    installer.pods_project.targets.each do |target|

        # Disable bitcode for each pod framework
        # Because the WebRTC pod (included by the JingleCallStack pod) does not support it.
        # Plus the app does not enable it
        target.build_configurations.each do |config|
            config.build_settings['ENABLE_BITCODE'] = 'NO'

            # Force SwiftUTI Swift version to 5.0 (as there is no code changes to perform for SwiftUTI fork using Swift 4.2)
            if target.name.include? 'SwiftUTI'
                config.build_settings['SWIFT_VERSION'] = '5.0'
            end
        end
    end
end
```

ℹ Please link to a project we can download that reproduces the issue.
https://github.com/vector-im/riot-ios

**Comment by dnkoutso** [comment]
Thanks for report and sample app, however, its still unclear from your repo what the actual issue is.

Can you please comment here and be more precise on what you want to accomplish?

**Comment by ZhangYiGang** [comment]
> Thanks for report and sample app, however, its still unclear from your repo what the actual issue is.
> 
> Can you please comment here and be more precise on what you want to accomplish?

I used two libraries in the open-source project riot (riot connects two open-source pod libraries, which are introduced in podsec). I downloaded them locally and wanted to add a bundle to them, but I succeeded in the matrix SDK and failed in the matrix kit. The statement I use is spec.resource_bundles = {'resourcebundle1' = > ['image1 / *. PNG ']}. Image is a folder under both library files, but in the generated framework, only the framework of matrix SDK has bundles, but not the framework generated by kit. Is it because there are the following words in the podspec file of kit？


s.subspec 'Core' do |core|
   
   core.resources = ["MatrixKit/**/*.{xib}","MatrixKit/Assets/resourceBundle.bundle", "MatrixKit/Assets/MatrixKitAssets.bundle"]
 end

Reiteration steps：
1. 
Download riot and two open source pod Libraries
riot:https://github.com/vector-im/riot-ios
matrixkit:https://github.com/matrix-org/matrix-ios-kit
matrixsdk:https://github.com/matrix-org/matrix-ios-sdk
Modify the matrixkitversion line of podfile to import the two libraries locally
2. Create a new image folder under two pod Libraries
3. Add this sentence to their podspec
spec.resource_bundles = {'resourcebundle1' = > ['image1 / *. PNG ']
4. pod install and build riot
5. After executing the above code, I found that there is no resourcebundle in one of the two generated frameworks（in  matrixkit）. In addition, the names of my two bundles are not the same. There should be no problem of resource conflict and import path, because image folders are in the first level of the project
So I want to ask, is it because of the difference between the podspec file of matrixkit and matrixsdk?
